### ISSUE TITLE
More fine-grained compilation stats

### ISSUE THREAD

**Issue by LebedevRI** [aras-p/ClangBuildAnalyzer#9]
Not sure if this is the correct place for this issue, but here it goes.
It would be good to see more detailed overview of the back-end phase.
In particular, it would be great to have at least global `**** Passes that took longest:`
Possibly something more fine-grained too (top-10 pass-function, pass-files pairs e.g.)

**Comment by LebedevRI** [comment]
Aha, and there already is OptFunction in OptPass; with RunPass'es that either a proper pass
(with a name in details), or containing RunLoopPass (also with name in details).
So this appears to be mostly the right place for this issue report.

**Comment by aras-p** [comment]
Yeah, will add that at some point. Busy with unrelated stuffs right now!
